karaf/features/src/main/feature/feature.xml

```diff
@@ -260,13 +260,13 @@
         <bundle dependency="true">mvn:org.apache.jclouds.driver/jclouds-okhttp/${jclouds.version}</bundle>
 
         <bundle dependency="true">mvn:org.apache.jclouds.driver/jclouds-sshj/${jclouds.version}</bundle>
-        <bundle dependency="true">mvn:org.apache.jclouds.labs/abiquo/${jclouds.version}</bundle>
-        <bundle dependency="true">mvn:org.apache.jclouds.labs/docker/${jclouds.version}</bundle>
+        <!--<bundle dependency="true">mvn:org.apache.jclouds.labs/abiquo/${jclouds.version}</bundle>--> <!-- jclouds abstraction not implemented -->
+        <bundle dependency="true">mvn:org.apache.jclouds.api/docker/${jclouds.version}</bundle>
 
-        <!-- Not bundle-ized, fixed in 2.0.0-SNAPSHOT -->
-        <!-- bundle dependency="true">mvn:org.apache.jclouds.labs/google-compute-engine/${jclouds.version}</bundle -->
+        <bundle dependency="true">mvn:org.apache.jclouds.provider/google-compute-engine/${jclouds.version}</bundle>
 
         <feature>jclouds-services</feature>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jsch-agentproxy-sshj/0.0.9_1</bundle>
```

Comment:
@andreaturli @googlielmo So I dont think we can get around including `net.schmizz.sshj` as [org.apache.servicemix.bundles.jsch-agentproxy-sshj](https://github.com/apache/servicemix-bundles/blob/master/jsch-agentproxy-sshj-0.0.9/pom.xml#L75-L79) and [jsch-agentproxy-sshj](https://github.com/ymnk/jsch-agent-proxy/blob/master/jsch-agent-proxy-sshj/pom.xml#L16-L20) depend upon it and there fore will not start. (Alos `jclouds-ssh` has `net.schmizz.sshj*` in its OSGi imports even though it excludes it explicity)

The good thing with the change to `com.hierynomus/sshj` is that nothing will import / use the old bundle as the symbolic name has changed.